labextension/src/widgets/deploys-progress/DeployProgress.tsx

```diff
@@ -105,6 +106,20 @@ export const DeployProgress: React.FunctionComponent<DeployProgress> = props =>
     );
   };
 
+  const getInfoBadge = (title: string, content: any) => {
+    return content ? (
```

Comment:
Let's convert this syntax to `<bool> && element`, so it's cleaner.
Take a look here https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator
I know that we basically never do this in the codebase, but I'd like to start introducing a cleaner syntax whenever possible :)